helm-chart/helm3/6.4.0/skywalking/README.md

```diff
@@ -111,7 +112,7 @@ The following table lists the configurable parameters of the Skywalking chart an
 | `elasticsearch.master.tolerations` | `[]` | Master tolerations |
 | `elasticsearch.master.heapSize` | `512m` | Master node heap size |
 | `elasticsearch.master.name` | `master` | Master component name |
-| `elasticsearch.master.persistence.enabled` | `true` | Master persistent enabled/disabled |
+| `elasticsearch.master.persistence.enabled` | `false` | Master persistent enabled/disabled |
```

Comment:
Why disable all of the persistence?

Reply:
If the storageClass is not set, an error will be returned.

Reply:
👍